client/src/pages/Bookings/Booking/Booking.tsx

```diff
@@ -0,0 +1,81 @@
+import { Grid, Stack, Typography, Paper } from '@mui/material';
+import AvatarDisplay from '../../../components/AvatarDisplay/AvatarDisplay';
+import { User } from '../../../interface/User';
+import EditBooking from './EditBooking';
+import { RequestStatus } from '../../../types/RequestStatus';
+
+interface Props {
+  start: string;
+  end: string;
+  userId: User;
+  requestId: string;
+  status: RequestStatus;
+  editable?: boolean;
+  border?: boolean;
+  handleRequest: (requestId: string, status: RequestStatus) => void;
+}
+
+export default function Booking({
+  start,
+  end,
+  userId,
+  requestId,
+  status,
+  editable = true,
+  border = false,
+  handleRequest: handleRequest,
+}: Props): JSX.Element {
+  // function renderRequestStatus(status: RequestStatus): string {
+  //   return status.toString();
+  // }
+
+  function parseStartDate(start: Date): string {
+    return start.toLocaleString('en-GB', { dateStyle: 'long', timeStyle: 'short', hourCycle: 'h12' });
+  }
+
+  function parseEndDate(start: Date, end: Date): string {
+    if (start.getDay() === end.getDay()) return end.toLocaleString('en-GB', { timeStyle: 'short', hourCycle: 'h12' });
+    return end.toLocaleString('en-GB', { timeStyle: 'short', hourCycle: 'h12' });
+  }
+
+  return (
+    <Paper {...(border ? { variant: 'outlined' } : {})} elevation={0} sx={{ marginY: 1 }}>
+      <Grid
+        container
+        direction="row"
+        justifyContent="flex-start"
+        alignItems="center"
+        paddingY={2}
+        paddingLeft={2}
+        paddingRight={2}
+        spacing={3}
+      >
+        <Grid item xs={9}>
+          <Typography sx={{ fontWeight: 'bold', fontSize: 16 }}>
```

Comment:
As opposed to using the `sx` prop to style `Typography` components inline, we can set 'font - weights and sizes' for variants in MUI's theme and use those variants throughout the app :)